package.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>reactive_planners</name>
+    <version>0.0.0</version>
+    <description>Reactive planners for locomotion of legged robots</description>
+    <maintainer email="stanley.wu@mail.mcgill.ca">Stanley Wu</maintainer><!-- Maintainer for anything related to ROS -->
+    <license>BSD-3-Clause</license>
+
+    <depend>mpi_cmake_modules</depend>
+    <depend>yaml_utils</depend>
```

Comment:
make sure you list all dependencies if you use such package.xml

make sure you add the build_tool you want use and export it:
```
    <!-- The following tags are recommended by REP-136 -->
  <exec_depend condition="$ROS_VERSION == 1">catkin</exec_depend>
  <exec_depend condition="$ROS_VERSION == 2">ament_cmake</exec_depend>


    <buildtool_depend>cmake</buildtool_depend>
    <export>
        <build_type>cmake</build_type>
    </export>
```